### PR TITLE
add basic runbook for production image errors and disaster recovery

### PR DESCRIPTION
**Pull request by listx**
Also add a production restoration script, which is the "inverse" of the
backup script.

/cc @thockin

### CODE REVIEW

**Comment by listx** [comment]
So, this runbook isn't much but it's better than nothing. Gotta start somewhere!

File: infra/gcp/runbook.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+If the manual promotion run above fails to reinstate all original images, then run:
+
+```
+$ <k8s.io_ROOT>/infra/gcp/backup_tools/restore_prod.sh
+```
+
+to restore production from the backups.
+
+## Deploy a new auditor image
```
Comment (alejandrox1) [inline infra/gcp/runbook.md:19]: Is there any documentation for the auditor image?

Also, how is the auditor image related to GCR images in general?
Does the auditor send any sort of information/alert in someplace?

**Comment by listx** [comment]
@alejandrox1 Sorry this took so long, but I added some more info.

**Comment by listx** [comment]
Ping?

**Pull request closed without merging**